doc/spelling.md

```diff
@@ -41,24 +41,45 @@ https://demo.noms.io/aa::music
 Value specifications take the form:
 
 ```
-<database>::<value-name>::<path>
+<database>::<value-name><path>
 ```
 
 See [spelling databases](#spelling-databases) for how to build the database part of the name.
 
-The `value-name` part can be either a hash or a dataset name. If  `value-name` matches the pattern `^#[0-9a-v]{32}$`, it will be interpreted as a hash. Otherwise it will be interpreted as a dataset name.
+The `value-name` part can be either a hash or a dataset name. If `value-name` matches the pattern `^#[0-9a-v]{32}$`, it will be interpreted as a hash otherwise it will be interpreted as a dataset name. See [spelling datasets](#spelling-datasets) for how to build the dataset part of the name.
 
-The `path` part is relative to the value at `value-name`. See [#1399](https://github.com/attic-labs/noms/issues/1399) for spelling.
+The `path` part is relative to the hash or dataset provided in `value-name`. 
+
+### Spelling Items in Structs
```

Comment:
How about making this heading something more direct (and ctrl+f-able) like: "specifying struct fields". And the below "indexing into collection"